Resources/Prototypes/silicon-laws.yml

```
@@ -542,14 +542,15 @@
 # ion storm random lawsets
 - type: weightedRandom
   id: IonStormLawsets
-  weights:
-    # its crewsimov by default dont be lame
+  weights: #mono - added Exodus & decreased Corp, Paladin, and 3L chances
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)